docs/src/components/Github/OrbitTeam.tsx

```diff
@@ -8,16 +8,12 @@ import Member from "./TeamMember";
 
 import { Contributor } from ".";
 
-interface AllMembers {
+interface Props {
   current: Contributor[];
   previous: Contributor[];
```

Comment:
I know these are old, but at least the interface would tell us it was about members. Now I think the readability would improve a lot if we could rename these to `currentMembers` and `previousMembers`, for example. WDYT?

Reply:
sure, makes sense, will rename ;)